consensus/ising/proposerservice.go

```diff
@@ -379,6 +380,8 @@ func (ps *ProposerService) TimeoutRoutine() {
 }
 
 func (ps *ProposerService) BlockPersistCompleted(v interface{}) {
+	ps.blockPersistedLock.Lock()
+	defer ps.blockPersistedLock.Unlock()
 	// reset timer when block persisted
 	ps.proposerChangeTimer.Stop()
 	ps.proposerChangeTimer.Reset(config.ProposerChangeTime)
```

Comment:
Is it possible that the ps.proposerChangeTimer.Reset call in ChangeProposerRoutine() is called at the same time with this?

Reply:
I considered it. the community met this bug in sync state. In sync state block will be persisted quickly.  proposerChangeTimer will be called when no block has be persisted  in one minute.